third_party/libunwindstack/include/unwindstack/Unwinder.h

```diff
@@ -66,7 +66,9 @@ class Unwinder {
   virtual ~Unwinder() = default;
 
   virtual void Unwind(const std::vector<std::string>* initial_map_names_to_skip = nullptr,
-                      const std::vector<std::string>* map_suffixes_to_ignore = nullptr);
+                      const std::vector<std::string>* map_suffixes_to_ignore = nullptr,
+                      const std::map<uint64_t /*function_start*/, uint64_t /*size*/>*
+                          functions_to_stop_at = nullptr);
```

Comment:
I was thinking: we have the comments with names, and we don't indicate that these are absolute addresses (we would need some comment somewhere).

Instead, what do you think of naming this parameter (in all the related functions) `absolute_address_to_size_of_functions_to_stop_at`? That wouldn't require any comment.

Reply:
Sound good. Done.